apps/web/src/app/layout.tsx

```diff
@@ -77,9 +78,16 @@ export default function RootLayout({
 }: {
   children: React.ReactNode
 }): React.JSX.Element {
+  const gaId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_MEASUREMENT_ID
+
+  if (!gaId || gaId === '') {
```

Comment:
Instead of this, add the variable declaration in here: https://github.com/keyshade-xyz/keyshade/blob/5f3fac830e3ef341ae239080ce09b67f1a6e0fc6/apps/platform/src/env.ts#L3-L5